package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "version": "2.0.0",
+  "version": "2.0.1",
```

Comment:
Should this version bump be higher? Maybe 2.1.0 if not 3.0.0

Reply:
This was a quick fix to a param name (the interface/functionality did not change). In the previous PR, we already incremented from v1 to v2.

Reply:
Oops! My bad. That makes perfect sense. Thanks!